mypy/stubgen.py

```diff
@@ -1396,6 +1397,8 @@ def is_private_member(self, fullname: str) -> bool:
     def get_str_type_of_node(
         self, rvalue: Expression, can_infer_optional: bool = False, can_be_any: bool = True
     ) -> str:
+        rvalue = self.unwrap_unary_expr(rvalue)
```

Comment:
I'm not sure we should be doing this with all possible `rvalue`s. Some unary expressions change the type:

```pycon
>>> type(-True)
<class 'int'>
```

Reply:
I guess it's pretty unlikely that this would ever come up in real code, but it would still be nice to be correct here imo :)